sweddit_combined/decisions/ChineseFormables.txt

```diff
@@ -1,143 +1,76 @@
 country_decisions = {
 
-	form_beiyang_china = {
-		major = yes
-		potential = {
-			NOT = { has_country_flag = formed_beiyang_china }
-			normal_or_historical_nations = yes
-			was_never_end_game_tag_trigger = yes
-			culture_group = east_asian
-			is_colonial_nation = no
-			NOT = { tag = IMP }
-		}
-		provinces_to_highlight = {
-			OR = {
-				province_id = 1816 #Beijing
-				province_id = 4194
-				province_id = 703 
-				province_id = 697
-				province_id = 2136
-				province_id = 696
-				province_id = 695
-				province_id = 2177
-				province_id = 693
-				province_id = 2178
-				province_id = 694
-				province_id = 692
-				province_id = 1836
-				province_id = 688
-				province_id = 2137
-				province_id = 2140
-				province_id = 691
-				province_id = 2138
-				province_id = 690
-				province_id = 2139
-			}
-			OR = {
-				NOT = { owned_by = ROOT }
-				NOT = { is_core = ROOT }
-			}
-		}
-		allow = {
-			adm_tech = 10
-			is_free_or_tributary_trigger = yes
-			culture_group = east_asian
-			is_at_war = no
-			owns_core_province = 1816 #Beijing
-			owns_core_province = 4194
-			owns_core_province = 703 
-			owns_core_province = 697
-			owns_core_province = 2136
-			owns_core_province = 696
-			owns_core_province = 695
-			owns_core_province = 2177
-			owns_core_province = 693
-			owns_core_province = 2178
-			owns_core_province = 694
-			owns_core_province = 692
-			owns_core_province = 1836
-			owns_core_province = 688
-			owns_core_province = 2137
-			owns_core_province = 2140
-			owns_core_province = 691
-			owns_core_province = 2138
-			owns_core_province = 690
-			owns_core_province = 2139
-		}
-		effect = {
-			1816 = {
-				add_base_tax = 2
-				add_base_production = 2
-				add_base_manpower = 2
-				move_capital_effect = yes
-			}
-			change_tag = RCH
-			remove_non_electors_emperors_from_empire_effect = yes
-			china_superregion = {
-				limit = {
-					NOT = { owned_by = ROOT }
-				}
-				add_permanent_claim = RCH
-			}
-			add_prestige = 20
-			set_government_rank = 2
-			if = {
-				limit = { has_custom_ideas = no }
-				country_event = { id = ideagroups.1 } #Swap Ideas
-			}
-			country_event = { id = warringchina.2 } #Swap Government
-			set_country_flag = formed_beiyang_china
-		}
-		ai_will_do = {
-			factor = 1
-		}
-		ai_importance = 400
-	}
-	
 	form_chinese_empire = {
 		major = yes
 		potential = {
 			NOT = { has_country_flag = formed_chinese_empire }
 			normal_or_historical_nations = yes
-			was_never_end_game_tag_trigger = yes
+			OR = {
+				tag = MNG
```

Comment:
Why is ming here?

Reply:
Ming är end game tag, så de får inte forma EoC, la in de så att de också kune